scripts/create_service.sh

```diff
@@ -26,8 +26,13 @@ fi
 
 case "${language}" in
     go)
-        build_script="cd \"traffic-director-grpc-examples/go/${service_type}_server\"
-sudo apt-get install -y golang
+        build_script="sudo apt-get install -y wget
+wget https://dl.google.com/go/go1.16.5.linux-amd64.tar.gz
+sudo tar -C /usr/local -xvf go1.16.5.linux-amd64.tar.gz
```

Comment:
What would tar do if `/usr/local/go` already exists?
Maybe we can install this go to `$HOME`, then update `PATH`

Reply:
The reason I found doing this useful is that if something went bad and you have to SSH into the VM to check things out, any updates done to `PATH` by the startup script are not reflected. So, putting `go` in `/usr/bin` which you are sure is part of `PATH` ensures that `go` command is available.

Reply:
> What would tar do if /usr/local/go already exists?

I could use one of these options:
```
       --overwrite
              Overwrite existing files when extracting.

       --overwrite-dir
              Overwrite metadata of existing directories when extracting
              (default).
```